docs/assets/diagrams/transfer/03-confirm-payee.puml

```diff
@@ -0,0 +1,118 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+Alice -> App: Initiate this transaction
+
+rnote right of App #Light
+  ""firebase.firestore()""
+  ""  .collection("transactions")""
+  ""  .doc("456")""
+  ""  .set({""
+  ""    sourceAccountId: "bob.fspb",""
+  ""    consentId: 123,""
+  ""    amount: {""
+  ""      amount: 20.00""
```

Comment:
I think this should technically be a string.

Reply:
The original intention was to give the mobile app flexibility to pass in a floating point value and let the server handle the rounding since Mojaloop [does not allow trailing zeros] in the amount field... but yeah after thinking more about this, it may also better to handle the conversion in the mobile app to reduce the amount of work needed by the server.

Also, can I check whether you store it as a number or string in the mobile app @MahidharBandaru ?